django/contrib/admin/widgets.py

```diff
@@ -500,6 +500,7 @@ def __init__(self, field, admin_site, attrs=None, choices=(), using=None):
         self.db = using
         self.choices = choices
         self.attrs = {} if attrs is None else attrs.copy()
+        self.option_attrs = {}
```

Comment:
Should we add a test in `admin_widgets/test_autocomplete_widget.py` where `option_attrs` are set?

Reply:
I'm not sure about this one 🤔 If I understood correctly, this change was added so it wouldn't break other tests, otherwise, it doesn't have any additional effect. Should we remove it from here and try to handle it differently?